.github/workflows/e2e-matrix.yml

```diff
@@ -44,15 +44,15 @@ jobs:
 
     steps:
     - name: Check out code onto GOPATH
-      uses: actions/checkout@v4
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
         path: ${{ github.workspace }}/src/github.com/tektoncd/pipeline
```

Comment:
@afrittoli any reason to checkout into GOPATH ? It shouldn't be needed nowadays (GOPATH I mean)

Reply:
`ko` does not work otherwise, last I tried. It may be different now.

Reply:
@afrittoli you mean `ko` on the github worklows ? Because it definitely works locally for me.

Reply:
Yes, but I didn't really try too hard to make it work, it just worked when checking out that way, so I kept it